mmdet/utils/misc.py

```diff
@@ -74,3 +75,15 @@ def update(cfg, src_str, dst_str):
 
     update(cfg.data, cfg.data_root, dst_root)
     cfg.data_root = dst_root
+
+
+_torch_version_div_indexing = (
+    'parrots' not in TORCH_VERSION
+    and digit_version(TORCH_VERSION) >= digit_version('1.8'))
+
+
+def floordiv(dividend, divisor, rounding_mode='trunc'):
+    if _torch_version_div_indexing:
```

Comment:
Actually, you can use `torch.floor_divide` directly

Reply:
[torch.floor_divide()](https://pytorch.org/docs/stable/generated/torch.floor_divide.html#torch.floor_divide) is deprecated and will be removed in a future PyTorch release.
https://pytorch.org/docs/stable/generated/torch.floor_divide.html